Sources/Kommunicate/Classes/KMPreChatFormViewController.swift

```diff
@@ -39,7 +39,7 @@ open class KMPreChatFormViewController: UIViewController {
         public init() {}
     }
 
-    public weak var delegate: KMPreChatFormViewControllerDelegate!
+    public var delegate: KMPreChatFormViewControllerDelegate!
```

Comment:
Don't remove `weak` otherwise there will be a strong reference which will result in memory leak.